pkgs/applications/science/math/sage/sagelib.nix

```diff
@@ -65,11 +65,13 @@ buildPythonPackage rec {
     perl
     jupyter_core
     pkg-config
+    pip # needed to query installed packages
   ];
 
   buildInputs = [
     gd
     readline
+    iml
```

Comment:
@FRidh I don't understand why I need to add `iml` to `buildInputs` to make the build succeed. Its already in `nativeBuildInputs`, yet the build fails without it also being in `buildInputs`.